src/main/scala/mesosphere/marathon/state/Migration.scala

```diff
@@ -364,9 +364,20 @@ class MigrationTo0_13(taskRepository: TaskRepository, store: PersistentStore) {
         _ <- store.delete(oldName)
       } yield ()
     }
-    store.load(oldName).flatMap {
-      case Some(key) => moveKey(key.bytes)
-      case None      => Future.successful(())
+
+    store.load(newName).flatMap {
+      case Some(_) =>
+        log.info("framework:id already exists, no need to migrate")
+        Future.successful(())
```

Comment:
you do not delete the "oldName" in this case, if it exists. But that shouldn't do any harm and actually I wouldn't know how to get into this situation, right?